docs/v3/core/v3.0.rst

```diff
@@ -1556,7 +1555,7 @@ Let "+" be the string concatenation operator.
 Storage transformers
 ====================
 
-A Zarr storage transformer allows to change the zarr-compatible data before storing it.
+A Zarr storage transformer allows a store to change zarr-compatible data before storing it.
```

Comment:
I am unsure about this change --- not sure if we consider a "storage transformer" a "store".  Everything else here looks good, though.

Reply:
i wasn't sure about this one either. Perhaps instead of "A Zarr storage transformer allows a Y to do X"  we can just say "A Zarr storage transformer does X", e.g. "A Zarr storage transformer modifies a request to store or retrieve data before passing that request to a store"

Reply:
> "A Zarr storage transformer modifies a request to read or write data before passing that request to a store"

This still refers to the next entitity after the transformer as a "store", but it could also be a storage transformer. So maybe
```suggestion
"A Zarr storage transformer modifies a request to read or write data before passing that request."
```
or
```suggestion
"A Zarr storage transformer modifies a request to read or write data before passing that request to the following transformer or store."
```